rd_ui/app/scripts/ng_highchart.js

```diff
@@ -325,6 +325,36 @@
                 series = seriesCopy;
               }
 
+              if (chartOptions['sortY'] === true || chartOptions['sortY'] === undefined) {
+                var sortable = true;
+
+                for (var i = 0; i < series.length; i++) {
+                  if (series[i].data.length != 1) {
```

Comment:
The goal of the sort-by-y functionality is to reorder different series in a bar chart. This only makes sense if there is one data point per series. Does that make sense, or are there cases which I haven't considered?